package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@festicket/eslint-config",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
Agree with the rule change but I don't think we should be bumping version numbers within PRs, this gets messed up if we have multiple concurrent PRs.